speak/models/README.md

```diff
@@ -1 +1,11 @@
-Place your model here
\ No newline at end of file
+Place your models here.
+
+## HuggingFace repositories
+
+This requires you to have `git lfs` installed.
+
+For example, to download CzeGPT-2, run
+
+```console
+git clone https://huggingface.co/MU-NLPC/CzeGPT-2
+```
```

Comment:
This is perhaps not necessary, the transformers library is able to pull the model itself if you use for example "MU-NLPC/CzeGPT-2" as the model path. 

One thing to mention though is that when using containers it might be necessary to pull the library again whenever the container is recreated or its image gets updated.

Reply:
I've dropped that commit. It should have been addressed separately anyway.